includes/reader-activation/sync/class-woocommerce.php

```diff
@@ -322,16 +319,16 @@ private static function get_order_metadata( $order, $payment_page_url = false )
 				$metadata['membership_status'] = $current_subscription->get_status();
 			}
 
-			$metadata['sub_start_date']      = $current_subscription->get_date( 'start' );
-			$metadata['sub_end_date']        = $current_subscription->get_date( 'end' ) ? $current_subscription->get_date( 'end' ) : '';
+			$metadata['sub_start_date']      = $current_subscription->get_date( 'start', 'site' );
+			$metadata['sub_end_date']        = $current_subscription->get_date( 'end', 'site' ) ?? '';
 			$metadata['billing_cycle']       = $current_subscription->get_billing_period();
 			$metadata['recurring_payment']   = $current_subscription->get_total();
 			$metadata['last_payment_amount'] = self::get_last_payment_amount( $current_subscription );
-			$metadata['last_payment_date']   = $current_subscription->get_date( 'last_order_date_paid' ) ? $current_subscription->get_date( 'last_order_date_paid' ) : gmdate( Metadata::DATE_FORMAT );
+			$metadata['last_payment_date']   = $current_subscription->get_date( 'last_order_date_paid', 'site' ) ?? current_time( Metadata::DATE_FORMAT );
```

Comment:
Not quite sure I understand why we'd use `gmdate()` or now `current_time()` if it it's empty (it could just be empty), but I'm just replicating the existing code.